torch_xla/test/test_utils.py

```diff
@@ -68,7 +68,9 @@ def get_summary_writer(logdir):
   """
   if logdir:
     from tensorboardX import SummaryWriter
-    return SummaryWriter(log_dir=logdir)
+    writer = SummaryWriter(log_dir=logdir)
+    write_to_summary(writer, 0, dict_to_write={'StartupTime': 0})
```

Comment:
Feels kinda weird that you post 0 as StartupTime.

Reply:
I can post `time.time()` instead?

Reply:
Feels better.

Reply:
Also change the name to be more explicit. `StartupTime` sounds like it could be how long some part of the system takes to start up